docs/api/tabs.md

```diff
@@ -24,17 +24,23 @@ The `ion-tabs` component does not have any styling and works as a router outlet
 
 Both `ion-tabs` and `ion-tab-bar` can be used as standalone elements. They don’t depend on each other to work, but they are usually used together in order to implement a tab-based navigation that behaves like a native app.
 
+:::warning
+Using `IonTabs` within React requires the `IonTabBar` component as a direct child of `IonTabs`.
+:::
```

Comment:
There is a separate [PR](https://github.com/ionic-team/ionic-framework/pull/29868) that removes the need to include `IonTabBar` within `IonTabs`. If that PR is approved first then I'll remove this warning before this docs PR is merged. Else I'll remove it in a separate PR.

Reply:
Should be good to remove this 👍